hack/lib/catalogsource.bash

```diff
@@ -1,22 +1,101 @@
 #!/usr/bin/env bash
 
-function ensure_catalogsource_installed {
-  logger.info 'Check if CatalogSource is installed'
-  if oc get catalogsource "$OPERATOR" -n "$OLM_NAMESPACE" >/dev/null 2>&1; then
-    logger.success 'CatalogSource is already installed.'
-    return 0
-  fi
-  install_catalogsource
-}
-
 function install_catalogsource {
   logger.info "Installing CatalogSource"
+  local rootdir="$(dirname "$(dirname "$(dirname "$(realpath "${BASH_SOURCE[0]}")")")")"
```

Comment:
Better to define local variable on separate line to avoid [SC2155](https://github.com/koalaman/shellcheck/wiki/SC2155)

Reply:
Copy & pasted from what was already there. Let's fix this afterwards if we feel it's important.

Reply:
In this exact case, not important. It's just good to know, and make a good habit, as it may save you a huge pain someday.